eks-hybrid-iam-ra-setup.sh

```diff
@@ -51,6 +69,20 @@ cat << EOF > "${SECRETS_DIR}/node.key"
 ${NODE_KEY_DATA}
 EOF
 
+if [ "${ENABLE_CREDENTIALS_FILE}" = "true" ]; then
+    EKS_HYBRID_SHARED_CREDENTIALS_FILE="/root/.aws/eks-hybrid/credentials"
```

Comment:
nit: this could be a constant at the top

Reply:
Fixed